excel.go

```diff
@@ -455,33 +455,30 @@ func updateMACFinUsers(f *excelize.File, input *Input, client S3ClientAPI, env E
 func protectExcelWorkbook(filename string) (string, error) {
 	dir := filepath.Dir(filename)
 	outFilename := filepath.Join(dir, "protected-"+filepath.Base(filename))
-	cat := exec.Command("cat", filename)
-	excel := exec.Command("secure-spreadsheet", "--password", os.Getenv("WORKBOOKPASSWORD"), "--input-format", "xlsx")
 
-	file, err := os.Create(outFilename)
+	_, err := exec.LookPath("secure-spreadsheet")
```

Comment:
Since you're validating the command exists, why not use the actual path to the executable in `exec.Command()` instead of throwing it away?

Reply:
I thought of that. My thought: It just makes the command longer. Easier to read with a short command name. The work the code does to lookup the command in the PATH seems trivial. I will change it to use the result of LookPath.